Assets/LeapMotion/Core/Scripts/XR/EyeType.cs

```diff
@@ -104,4 +105,4 @@ public enum OrderType {
       _hasBegun = false;
     }
   }
-}
+}
```

Comment:
You _removed_ the newline at the end of the file? ಠ_ಠ

Reply:
Not _intentionally_ >_<